app/src/main/java/com/globant/weatherapp/mvp/view/CityView.kt

```diff
@@ -3,20 +3,27 @@ package com.globant.weatherapp.mvp.view
 import kotlinx.android.synthetic.main.activity_welcome_layout.activity_welcome_autocomplete_text_view
 import android.app.Activity
 import android.widget.ArrayAdapter
-import android.R
+import android.widget.Toast
+import com.globant.weatherapp.R
 import com.globant.weatherapp.activities.WeatherInfoActivity
 import com.globant.weatherapp.mvp.contracts.CityContracts
 import com.globant.weatherapp.mvp.view.base.ActivityView
+import com.globant.weatherapp.utils.Constants.Companion.EMPTY_STRING
 
 class CityView(activity: Activity) : ActivityView(activity), CityContracts.View {
     override fun initView(cityNameList: MutableList<String>) {
         activity?.activity_welcome_autocomplete_text_view?.
-            setAdapter(ArrayAdapter(context, R.layout.simple_dropdown_item_1line, cityNameList))
+            setAdapter(ArrayAdapter(context, android.R.layout.simple_dropdown_item_1line, cityNameList))
```

Comment:
First, rename the layout . Second, by change the R import ?

Reply:
This layout is native from android, I use it for the card views, I had put like that because after in the same class I use the R of my app to use the string resources and if I use R.xxx in both lines, it throws errors. So I decided to put only one in the imports.